.github/workflows/ci.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       - uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
+          extensions: gmp
```

Comment:
@anavarre Is this available in Cloud IDE?

Reply:
Yes!

```
test:~/project (pipelines-build-master +) $ php -m | grep gmp
gmp
```